terragraph_planner/los/helper.py

```diff
@@ -966,7 +966,7 @@ def add_links_to_topology(
                                 rx_site.device.device_sku,
                             )
                         ]
-                        and link.el_dev <= max_el_scan_angle
+                        and abs(link.el_dev) <= max_el_scan_angle
```

Comment:
nit: Maybe we'd better update the comment for the max_el_scan_angle in the template config.